### DISCUSSION TITLE
NeoVim + Device Activation

### DISCUSSION THREAD

**Discussion by exAspArk** [community/community#7287] [Copilot Conversations] [answered]
Hey - I'm trying to use https://github.com/github/copilot.vim. After running `:Copilot setup`, it opens my browser and asks for a device activation code:

![image](https://user-images.githubusercontent.com/577441/140336841-196a1fe4-fcd6-4177-a4ec-6cf6de86c04b.png)

How do I get the code? Is there any other way to authenticate? 

I don't use VS Code or GH CLI. I couldn't find any instructions in the docs https://github.com/github/copilot-docs

cc @tpope

**Comment by tpope** [comment, marked as answer]
When you invoke `:Copilot setup`, you'll get output that looks like this:

```
First copy your one-time code: XXXX-XXXX
Press ENTER to open https://github.com/login/device in your browser
```

The `XXXX-XXXX` is the code.

**Reply by exAspArk** [reply]
Oh, I see. I wasn't paying attention to the previous line. Thank you!
